index.html

```diff
@@ -79,24 +87,24 @@ <h2 class="multi-post">Multi-Post Stories</h2>
           <li class="list">Ruby</li>
          </ul>
          <button class="project" id="project" type="button">See Project</button>
+         </div>
         </div>
-      </div>
-
-      <div class="professional">
-        <div class="pro-art">
+     <div class="cards">
+      <div class="professional" id="des-img1">
+        <div class="pro-art" id="img1-disc">
          <h2 class="art-print">Profesional Art Printing Data</h2> 
           <p class="daily-selection">A daily selection of privately personalized reads;
             no accounts or sign-ups required. has been the industry's standard
           </p>
         </div>
-        <ul class="pro-tag">
+        <ul class="pro-tag" id="img1-tag">
           <li class="pro-h">html</li>
           <li class="pro-h">bootstrap</li>
           <li class="pro-h">Ruby</li>
         </ul>
-           <button class="see-proj" type="button">See Project</button>
+           <button class="see-proj" id="see-btn" type="button">See Project</button>
       </div>
```

Comment:
- [ ] Kindly fix the spacing of the buttons and technologies for the works section to match the Figma template, please.
## Your design
![r2](https://user-images.githubusercontent.com/94551484/186989131-04276510-3a20-43d4-84cb-d37d1bebf074.png)
## Figma template
![s2](https://user-images.githubusercontent.com/94551484/186989147-63563b13-2c87-41a1-bfe3-dc3662446e5e.png)

Reply:
I kindly advise you to follow the previous reviewer's advice.